### PR TITLE
Adds the Developer Certificate of Origin and Contributors

### PR DESCRIPTION
**Pull request by Weltraumschaf**
Adds the Developer Certificate of Origin and Contributors.

### CODE REVIEW

**Review by rseedorff — APPROVED** [review]

File: CONTRIBUTORS.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+```text
+Signed-off-by: gordon shumway <alf@melmak.com>
+```
+
+Committing with `git commit -s` will add the sign-off at the end of the commit message automatically for you.
+
+## List of Contributors
+
+- Sven Strittmatter <sven.strittmatter@iteratec.com>
```
Comment (fuhrmeistery) [inline CONTRIBUTORS.md:19]: Maybe it would make sense to discriminate between contributor and maintainer?

Comment (Weltraumschaf) [reply CONTRIBUTORS.md:19]: Have have no idea :-)

Didn't found any example for that. In the first place adding your self here is like "signing the fax" in the CLA process. So, IMHO we need not to mind it here at the moment.

Comment (fuhrmeistery) [reply CONTRIBUTORS.md:19]: I don't remember the repo that I had open at this point, but Ansible has separate files for maintainers and contributors 
See https://github.com/ansible-collections/community.postgresql

Comment (J12934) [reply CONTRIBUTORS.md:19]: I don't understand what this list is for?
Is this a "Hall of Fame" style of list, or is this required for the DCO part?

Comment (Weltraumschaf) [reply CONTRIBUTORS.md:19]: They just use plain text files with the GH handle and they do not use DCO anyway. So I'm not convinced why this is useful.

I suggest to move this discussion into the related issue and merge this so we can sign the DCO.

**Review by fuhrmeistery — APPROVED** [review]

**Pull request merged**